tests/benchmarks.py

```diff
@@ -0,0 +1,61 @@
+def bench_prime_generation():
```

Comment:
Can you put this in a new subdirectory `benchmarks` instead of `tests`? You'll also need to create a blank `benchmarks/__init__.py`. We will eventually want to run tests automatically after each push, but not benchmarks. We should also consider making the tests much faster by ditching the 5000 test (or move it to benchmarks).